hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -92,7 +92,7 @@
       desc: '''
             Verify that the DUT handles the NIST test vectors correctly.'''
       milestone: V2
-      tests: []
+      tests: [aes_nist_vectors]
```

Comment:
Hey @rasmus-madsen,
thanks for doing this. You'll have to put quotes around the test name: aes_nist_vectors -> "aes_nist_vectors".
Otherwise it doesn't work (CI is failing).

Reply:
Thanks - I am not sure how I missed this :D